config/initializers/chunked_uploader.rb

```diff
@@ -1,2 +1,4 @@
 # License: AGPL-3.0-or-later WITH Web-Template-Output-Additional-Permission-3.0-or-later
-CHUNKED_UPLOADER = ENV['CHUNKED_UPLOAD_CLASS'] ? ENV['CHUNKED_UPLOAD_CLASS'].constantize : ChunkedUploader::S3
\ No newline at end of file
+Rails.application.config.after_initialize do
+  CHUNKED_UPLOADER = ENV['CHUNKED_UPLOAD_CLASS'] ? ENV['CHUNKED_UPLOAD_CLASS'].constantize : ChunkedUploader::S3
```

Comment:
Rails 6 doesn't like some constants being set outside of an after_initialize block. Not sure why, but it gave me a warning on this one.